Tests/SnapshotTests/Extensions/SwiftUI+XCTestCase+Extensions.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Created by Backbase on 29/02/2024.
+//
+
+import SwiftUI
+import XCTest
+import BackbaseDesignSystem
+
+extension XCTestCase {
+
+    func convertToUIView(from someView: some View) -> UIView {
```

Comment:
this is for swiftUI specifically, not sure if we have in the golden sample app but if not we should remove this for now probably

Reply:
done, removed!